frontend/src/app/shared/community-agreement/community-agreement.widget.html

```diff
@@ -0,0 +1,209 @@
+<mat-card-content class="content-container">
+  <p *ngIf="!has_user_agreed" style="margin-left: 15px; margin-top: 10px">
+    ***Please read and accept the community agreement below to continue to CSXL
+    web page.***
+  </p>
+  <mat-card-header>
+    <h1>
+      Undergraduate Computer Science Experience Labs (XL) Community Standards
+      Agreement
+    </h1>
+  </mat-card-header>
+  <mat-card-content>
+    <p>
+      The Undergraduate Experience Labs (XL) community serves the Computer
+      Science Undergraduate Community at UNC-Chapel Hill.
+    </p>
+    <p>
+      The XL is dedicated to providing a harassment-free and inclusive
+      experience for everyone regardless of gender identity and expression,
+      sexual orientation, disabilities, neurodiversity, physical appearance,
+      body size, ethnicity, nationality, race, religion, or other protected
+      category. These policies are inspired by and almost entirely replicate
+      <a
+        href="https://developers.google.com/community-guidelines"
+        target="_blank"
+        style="color: rgb(67, 140, 200)"
+        >Google's Developer Community Guidelines.</a
+      >
+    </p>
+    <p>
+      We do not tolerate harassment of XL community members in any form. We take
+      violations of our policy seriously and will respond appropriately.
+    </p>
+    <p>
+      All participants of the XL in-person and on-line communities, XL
+      ambassadors, XL staff, speakers, faculty, and participants otherwise, must
+      agree to and abide by the following policies in order to access XL's
+      events, online communities, and facilities.
+    </p>
+    <h3>I agree to be respectful to everyone in the XL community.</h3>
```

Comment:
It's currently a little too easy to just blow throw the reading of this agreement like a typical EULA and not read anything. Can we make the case where someone _has not agreed_ include check boxes that must be checked for each of these `h3` elements? In the `Accept` button case, we can check to be sure each has been checked and ask the user to agree to all community expectations if they did not check any.